mountpoint-s3/Cargo.toml

```
@@ -7,7 +7,7 @@ publish = false
 default-run = "mount-s3"
 
 [dependencies]
-fuser = { path = "../mountpoint-s3-fuser", version = "0.15.0", features = ["abi-7-28"] }
+fuser = { path = "../mountpoint-s3-fuser", package = "mountpoint-s3-fuser", version = "0.1.0", features = ["abi-7-28"] }
```

This can just be:
```suggestion
mountpoint-s3-fuser = { path = "../mountpoint-s3-fuser", version = "0.1.0", features = ["abi-7-28"] }
```

You have already configured the new `mountpoint-s3-fuser` crate to export the `fuser` lib.